assets/sass/_aesop.scss

```diff
@@ -326,8 +337,55 @@
   }
 }
 
-.aesop-sticky-map.aesop-sticky-map-left .aesop-map-component:before {
-  box-shadow: none;
+.aesop-sticky-map {
+  .aesop-map-component:before {
+    box-shadow: none;
+  }
+
+  .header {
+    left: 0;
+    right: 0;
+  }
+
+  @include min-width(767px) {
```

Comment:
Expected item on line 350 to appear before line 341. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets